recipe/meta.yaml

```diff
@@ -62,6 +62,7 @@ outputs:
     script: build_base.bat  # [win]
```

Comment:
This recipe could be simplified by having diffusers-torch and diffusers as meta packages.
Here we are building the same thing thrice.
E.g.:
```
  - name: diffusers-torch
    build:
      skip: true  # [py>310 and ppc64le]
    requirements:
      host:
        - python
      run:
        - python
        - {{ pin_subpackage(name+'-base', exact=True) }}
        - pytorch >=1.4
        - huggingface_accelerate >=0.11.0
    test:
      imports:
        - diffusers
        - diffusers.models
      commands:
        - pip check
        - diffusers-cli --help
      requires:
        - pip
```